monkey/agent_plugins/payloads/ransomware/src/ransomware_options.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from pydantic import Field, validator
+
+from common.base_models import InfectionMonkeyBaseModel
+
+from .options_validators import (
+    VALID_FILE_EXTENSION_REGEX,
+    VALID_RANSOMWARE_PATH_LINUX_REGEX,
+    VALID_RANSOMWARE_PATH_WINDOWS_REGEX,
+)
+
+
+class EncryptionBehavior(InfectionMonkeyBaseModel):
+    file_extension: str = Field(
```

Comment:
```suggestion
    file_extension: constr = Field(
```

This should be constrained by `VALID_FILE_EXTENSION_REGEX`.